docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,219 @@
+
+MC is the component which glues/integrates/orchestrates these services. MC is the orchestrator in the context of Mobile.
+
+As well as the integration of Mobile Services with each other, another key aspect of building a solution is the integration with the platform, which is OpenShift. MC takes ownership of this aspect of the solution design also.
+
+== Design Principles
+
+MC interacts with OpenShift and the Kubernetes Core and not directly with the Mobile Services.
+
+== MC Installation
+The MC makes the Openshift platform Mobile aware. To do this the MC must perform a number of actions as part of its installation process.
+
+
+* *Step 1*: Ensure that Mobile Services are made available through the Service Catalog
+
+For mobile developers to be able to consume Mobile Services, they must be available to them. The medium through which services are presented is the https://docs.openshift.com/container-platform/3.7/architecture/service_catalog/index.html[Openshift Service Catalog]. In order to achieve this, the MC must perform a number of tasks:
+
+** *Step 1.1*: Install / Configure an OpenShift Ansible Broker (OAB)
++
+The Mobile Services are represented as Ansible Playbooks, https://github.com/ansibleplaybookbundle/ansible-playbook-bundle[APB]. For these playbooks to be read in, and represented through the Service Catalog, an https://docs.openshift.com/container-platform/3.7/architecture/service_catalog/ansible_service_broker.html#service-catalog-spec-file[Ansible Service Broker] must be created.
++
+image:images/5.x-Architecture-MC-ASB.png["My screenshot",width=1280, caption="Figure 1: Mobile Core Installation - Integrating Mobile Services into Service Catalog", title="Mobile Core Installation - Integrating Mobile Services into Service Catalog", link="images/5.x-Architecture-MC-ASB.png"]
++
+When the MC is installed, it will register a Broker with the Service Catalog. In some cases an OAB may already exist. A service broker is a server that conforms to the https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md[OSB API] specification and manages a set of one or more services. The software could be hosted within your own OpenShift Container Platform cluster or elsewhere. The MC will create a specific type of Broker, an Ansible Service Broker. Once the ASB is created, it will try and invoke a GET on the its configured endpoint, to retrieve the catalog of services it has access to. The set of services are defined through Ansible playbooks, which for Mobile are located in https://github.com/aerogearcatalog[AeroGear Docker Hub]. These services are returned to the Service Catalog and  https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/resources.md[Cluster Service Class] resources are created to represent each returned service type. At that point a user can see a representation of the services inside the Service Catalog.
```

Comment:
This paragraph is a little off. We switch between using the term OAB and ASB but these are actually the same things, To me using ASB throughout would make the most sense.
Also further down it says:
```
 Once the ASB is created, it will try and invoke a GET on the its configured endpoint, to retrieve the catalog of services it has access to.

```
Actually it is the service catalog that invokes the broker to return the services.


I think
```
At that point a user can see a representation of the services inside the Service Catalog.
```
should be ```At that point a user can see a representation of the services inside the Service Catalog UI.```